src/types/stan.ts

```diff
@@ -297,12 +297,13 @@ export type HistoryProps = HistoryStrProps | HistoryNumProps;
  */
 export const statusSort = (rowAStatus: WorkStatus, rowBStatus: WorkStatus) => {
   const statusArray: WorkStatus[] = [
+    WorkStatus.Unstarted,
```

Comment:
These changes should be reverted as this is the method used for standard dashboard and needs a particular order.